packages/shared/package.json

```diff
@@ -32,7 +32,7 @@
     "@linzjs/metrics": "^6.32.1",
     "aws-sdk": "^2.890.0",
     "entities": "^4.3.0",
-    "pino": "^8.3.0",
+    "pino": "^8.6.1",
```

Comment:
you need to upgrade all versions of pino no this specific one, this is making it so we now have multiple versions of pino installed.

Reply:
I think that's the only `pino` in Basemaps. I think the other pino are come from different project like `@linzjs/lambda`.

Reply:
I tried to run `yarn upgrade pino` seems not changing `yarn.lock` at all.